modal/cls.py

```diff
@@ -139,7 +139,7 @@ def _deps():
             # ugly - needed for .local()  TODO (elias): Clean up!
             partial_function.raw_f,
             user_cls=cls._user_cls,
-            serialized=service_function.info.is_serialized(),
+            serialized=True,  # service_function.info.is_serialized(),
```

Comment:
This is pretty stupid, but since service_function.info isn't defined for the "synthetic" local function that's created in the container entrypoint, this crashes without removing the serialized-value. 

As the TODO above says, we don't actually need a `FunctionInfo` here in the sense that we don't `load()` these functions (which is what it's made for). It just happens that we use `.info.raw_f` when doing .local() calls too, which makes `.info` a requirement just as a vessel for `.raw_f` and `user_cls` :(

With some refactoring of `.local()` we could remove this. Would be nice to not hold up this PR on that refactor though...